clang/test/3C/allarrays.c

```diff
@@ -1,4 +1,5 @@
-// RUN: 3c -alltypes %s -- | FileCheck -match-full-lines --check-prefixes="CHECK" %s
+// RUN: 3c -alltypes %s | FileCheck -match-full-lines --check-prefixes="CHECK" %s
+// RUN: 3c -alltypes %s | %clang -c -f3c-tool -fcheckedc-extension -x c -o /dev/null -
```

Comment:
I think we discussed dropping use of `/dev/null` entirely and using `%t` instead? Doing so is to enhance Windows compatibility. This way, we would be indicating this is a temp file, it will get removed by FileCheck automatically, right?

Reply:
I didn't realize it was more than a bugfix when piping on windows. But this seems reasonable. I'll change it and add it to the other list of test changes.

Reply:
I don't think `/dev/null` is a problem here. `llvm-lit` is supposed to automatically substitute temp files for `/dev/null` on Windows, but was not doing this correctly when dealing with pipes and io redirection.